bazarr/subtitles/upgrade.py

```diff
@@ -218,7 +228,7 @@ def upgrade_subtitles():
                     result = result[0]
                 store_subtitles_movie(movie['video_path'],
                                       path_mappings.path_replace_movie(movie['video_path']))
-                history_log_movie(3, movie['radarrId'], result, upgraded_from_id=movie['id'])
+                history_log_movie(3, movie['radarrId'], result, upgraded_from_id=movie['original_id'])
```

Comment:
It would be nice to create enum constants for magic numbers like "3".
I have no idea what "3" means here.

Reply:
3 is upgrade but yes it would be a good idea.